src/Softeq.NetKit.Chat.Web/Controllers/MemberController.cs

```diff
@@ -120,5 +121,46 @@ public async Task<IActionResult> UpdateUserNotificationSettingsAsync(Notificatio
         }
 
         #endregion
+
+        #region Notifications
+        
+        [ProducesResponseType(typeof(void), StatusCodes.Status200OK)]
+        [HttpPost]
+        [Route("push-token/subscribe")]
+        public async Task<IActionResult> SubscribePushToken([FromBody]PushTokenRequest model)
+        {
+            var userId = GetCurrentSaasUserId();
+
+            var token = model.Token.Trim().Replace(" ", string.Empty);
```

Comment:
Why we use Trim() here?

Reply:
because we are faced with spaces in device token received from mobile